src/components/DiffView/index.tsx

```diff
@@ -13,24 +22,81 @@ class DiffView extends React.Component<Props> {
     viewType: 'unified',
   };
 
-  render = () => {
+  renderHeader({ hunks }: DiffInfo) {
+    const { additions, deletions } = hunks.reduce(
+      (acc, hunk) => {
+        return {
+          additions:
+            acc.additions +
+            hunk.changes.filter((change) => change.isInsert).length,
+          deletions:
+            acc.deletions +
+            hunk.changes.filter((change) => change.isDelete).length,
+        };
+      },
+      { additions: 0, deletions: 0 },
+    );
+
+    return (
+      <div className={styles.header}>
+        <div className={styles.stats}>
+          <span className={styles.statsAdditions}>{`+++ ${additions}`}</span>
+          <span className={styles.statsDeletions}>{`--- ${deletions}`}</span>
+        </div>
+      </div>
+    );
+  }
+
+  renderHunk(hunk: HunkInfo, index: number) {
```

Comment:
Since this is used as a callback to `.map()` it might make more sense to define it as a bound function to avoid future surprises.

Reply:
Why not, although `renderHunks` is bound already, no?

Reply:
I think accessing `this` wouldn't work in the way you last had it.